### PR TITLE
fix(format-number): round to one decimal

### LINKED ISSUE #1455: crdt racial dash multiplier rounding
Right now, the headline for the viz goes out to four sf. Let's reduce that to just one decimal
image

### PR DESCRIPTION
**Pull request by schwartzadev**
Fixes #1455

### CODE REVIEW

File: src/components/utils/format.js [hunk 1]
```diff
@@ -31,9 +31,11 @@ const FormatDate = ({
   )
 }
 
-const FormatNumber = ({ number, nullValue = false }) => (
-  <>{number || number === 0 ? number.toLocaleString() : nullValue || 'N/A'}</>
-)
+const FormatNumber = ({ number, nullValue = false }) => {
```
Comment (kevee) [inline src/components/utils/format.js:34]: @schwartzadev what if we make the option for precision a prop for `FormatNumber`?

**Review by kevee — CHANGES_REQUESTED** [review]
I think we might want to make precision a prop so we can re-use this feature and also existing implementations won't get rounded.

File: src/components/utils/format.js (hunk 1 above)
Comment (schwartzadev) [reply src/components/utils/format.js:34]: Good call!

**Review by kevee — APPROVED** [review]
LGTM! Thanks!

**Pull request merged**